nuxeo-distribution/nuxeo-server-tomcat/src/main/resources/tomcat/bin/nuxeoctl.bat

```diff
@@ -208,7 +210,7 @@ if "%JAVA_VERSION%" lss "%REQUIRED_JAVA_VERSION%" (
   goto END
 )
 
-if "%JAVA_OPTS%" == "" set JAVA_OPTS=-Xms512m -Xmx1024m -Djava.net.preferIPv4Stack=true -Dsun.rmi.dgc.client.gcInterval=3600000 -Dsun.rmi.dgc.server.gcInterval=3600000 -Dfile.encoding=UTF-8
+if "!JAVA_OPTS!" == "" set JAVA_OPTS=-Xms512m -Xmx1024m -Djava.net.preferIPv4Stack=true -Dsun.rmi.dgc.client.gcInterval=3600000 -Dsun.rmi.dgc.server.gcInterval=3600000 -Dfile.encoding=UTF-8
```

Comment:
This is the fix of the ticket, other lines are improvements (can be removed if they're confusing). In a string comparison, if one of the strings to be compared contains a double quote (`"`), the variable expansion throws an error.

Example:

```bat
@echo off
SetLocal

set Lhs="
set Rhs="

if "%Lhs%" == "%Rhs%" echo Equal
```

Error:

```
echo was unexpected at this time.
```

The solution is to enable delayed expansion and use `!` instead of `%`. See [this post](https://stackoverflow.com/a/26079381/4381330) for more detail.